src/components/VideosModal.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import PropTypes from "prop-types";
+import {Collapsible, CollapsibleItem, Icon } from 'react-materialize';
+
+export default function VideosModal({ setVideoModal }) {
+  return (
+    <div id="modal" className="modal-container">
+      {/* React Materalize import */}
+      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css"></link>
```

Comment:
I had no idea we could link Materialize into just one file. I thought it was all or nothing. 😆  Clever!

Reply:
neither did we untill we tried it haha!